README.md

```diff
@@ -59,11 +59,11 @@ configurable options as to how those notifications are to be received. `pinax-no
 
 #### Supported Django and Python versions
 
-Django / Python | 3.5 | 3.6 | 3.7 | 3.8
---------------- | --- | --- | --- | ---
-1.11 |  *  |  *  |  *  |  *  
-2.x  |  *  |  *  |  *  |  *
-3.0  |     |  *  |  *  |  *
+Django \ Python | 3.6 | 3.7 | 3.8
```

Comment:
@KatherineMichel why the swap of slash direction here?

Reply:
I must have accidentally copied it from someplace that had it backwards.